client_lib/pulp/client/commands/repo/importer_config.py

```diff
@@ -96,6 +97,12 @@ def __init__(self):
             parse_func=parsers.pulp_parse_optional_nonnegative_int
         )
 
+        d = _('content downloading policy ({m})'.format(
+            m=' | '.join((constants.DOWNLOAD_IMMEDIATE,
+                          constants.DOWNLOAD_BACKGROUND,
+                          constants.DOWNLOAD_ON_DEMAND))))
+        self.opt_download_policy = PulpCliOption('--download-policy', d, required=False)
```

Comment:
Since there is a known set of valid values, it'd be nice to have a validator on the option.